guides/operation/auditing/log-webhooks-datadog.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Sending Audit Events to Datadog
+description: Configure Flipt to send audit event logs to Datadog
+---
+
+As described in the [Auditing Overview](/configuration/auditing/overview) configuration section, Flipt supports the ability to send audit events to configured sinks.
+
+This guide will cover how to configure Flipt to send audit events to Datadog using our [Webhook Sink](/configuration/auditing/webhooks) along with our templating feature.
+
+## What You'll Learn
+
+- 📊 How to create an API key in Datadog
+- 📡 How to configure Flipt to send audit events to Datadog
+
+## Prerequisites
+
+For this guide, you will need:
+
+- A [Datadog](https://datadoghq.com) account
+- A Datadog plan that supports [Log Management](https://docs.datadoghq.com/logs/)
+- [Docker](https://www.docker.com/)
+
+## API Configuration in Datadog
+
+### Step 1: Generate an API Key
+
+To send logs to Datadog, you will need to create an API key in your Datadog account.
+
+![Datadog API Keys](/images/guides/operation/log-webhooks-datadog/datadog-api-keys.png)
+
+1. Navigate to the [API Keys](https://app.datadoghq.com/organization-settings/api-keys) section of your Datadog organization settings.
+1. Click on the `+ New Key` button.
+1. Enter a name for your API key and click `Create Key`.
+1. Copy the API key somewhere safe, as you will need it later.
+
+![Datadog Create API Key](/images/guides/operation/log-webhooks-datadog/datadog-new-api-key.png)
+
+### Step 2: Find your Datadog Log Ingestion URL
+
+To send logs to Datadog, you will need to find your Log Ingestion URL.
+
+1. Navigate to the [Send Logs](https://docs.datadoghq.com/api/latest/logs/?code-lang=curl&site=us5#send-logs) documentation for the Datadog API.
+1. Copy the URL under the `Send logs` section. Save this URL, as you will need it later.
+
+![Datadog Log Ingestion URL](/images/guides/operation/log-webhooks-datadog/datadog-logs-api.png)
+
+## Configuring Audit Events in Flipt
+
+### Step 1: Define a Flipt `config.yml`
+
+1. Define a configuration file `config.yml` with the following configuration:
+
+```yaml
+audit:
+  sinks:
+    webhook:
+      enabled: true
+      templates:
+        - url: "< Datadog log ingestion URL >"
+          headers:
+            Content-Type: "application/json"
+            Content-Encoding: "gzip"
+            Authorization: "DD-API-KEY < Datadog API key >"
+          body: |
+            {
+              "type": "{{ .Type }}",
+              "action": "{{ .Action }}",
+              "payload": {{ toJson .Payload }}
+            }
+```
```

Comment:
@markphelps Having such a guide would be great.

I have a follow-up question about the body data since I'm unable to verify it myself. In the Datadog API documentation, all the example logs are sent as arrays. Additionally, there are a few reserved attributes that could potentially aid in the aggregation console. Here’s an example from their site:

```json
[
  {
    "ddsource": "nginx",
    "ddtags": "env:staging,version:5.1",
    "hostname": "i-012345678",
    "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello World",
    "service": "payment"
  }
]
``` 
Probably we could have `ddsource` set to `flipt` in the guide.